core/drivers/firewall/stm32_rifsc.c

```diff
@@ -157,6 +160,85 @@ struct rifsc_platdata {
 	struct risal_cfg *risal;
 	unsigned int nrisal;
 	bool is_tdcid;
+	bool errata_ahbrisab;
+};
+
+/**
+ * struct rimu_risup_pairs - Association of a RISUP and RIMU ID for a peripheral
+ *
+ * @rimu_id: ID of the RIMU
+ * @risup_id: ID of the associated RISUP
+ */
+struct rimu_risup_pairs {
+	uint32_t rimu_id;
+	uint32_t risup_id;
+};
+
+static const struct rimu_risup_pairs rimu_risup[] = {
+	[0] = {
+		.rimu_id = 0,
```

Comment:
Field `rimu_id` seems to always be equal to the index of the related cell in `rimu_risup`. Will it be needed in the long term for specific IDs?

Reply:
Yes, for the long term